src/main/java/core/ApplicationMojo.java

```diff
@@ -0,0 +1,20 @@
+package core;
+
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+
+import java.io.IOException;
+
+@Mojo(name = "start")
+public class ApplicationMojo extends AbstractMojo {
+    @Override
+    public void execute() throws MojoExecutionException {
+        try {
+            Application.main(new String[0]);
+        } catch (IOException e) {
+            throw new MojoExecutionException("Error while executing ADCL application", e);
```

Comment:
Too vague, please specify message so that the user can comprehend, why the plugin crashed.

Reply:
Nicht möglich, steht ja auch im cause dabei. Einziges was möglich wäre wäre `e.getMessage()` mit einzutragen. Dies ist aber redundant, da ja im cause schon dabei und damit mit ausgegeben.

Reply:
Vielleicht aber besser in der Errormessage zu beschreiben, wo es fehlschlägt
```suggestion
            throw new MojoExecutionException("Error while executing ADCL start MOJO", e);
```